saffron/src/lib.rs

```diff
@@ -3,6 +3,7 @@ pub mod cli;
 pub mod commitment;
 pub mod diff;
 pub mod env;
+pub mod read_proof;
```

Comment:
Please, see comment regarding git history.

Reply:
Yep! I'm planning to squash everything into one commit before merging, intermediate commits do not make individual sense anyway.